pkg/apis/v1beta1/api.go

```diff
@@ -27,6 +27,8 @@ import (
 // APISpec ...
 type APISpec struct {
 	Address         string            `yaml:"address"`
+	Port            int               `yaml:"port"`
+	K0sAPIPort      int               `yaml:"k0s_api_port,omitempty"`
```

Comment:
hmm, I'm not a huge fan of the underscored yaml key "scheme". Why not use camel casing as with all other multi-word keys?

Reply:
changed